lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```diff
@@ -129,6 +129,17 @@
     margin-top: $base-spacing;
   }
 
+  .social-share-bar {
```

Comment:
We should make this into a pattern. Maybe move to `patterns/` directory and `@TODO` it?